addons/l10n_in_edi/views/res_config_settings_views.xml

```diff
@@ -31,6 +31,9 @@
                             <div class='mt8'>
                                 <button name="l10n_in_edi_test" icon="fa-arrow-right" type="object" string="Verify Username and Password" class="btn-link"/>
                             </div>
+                            <div class='mt8'>
+                                <button name="buy_iap_l10n_in_edi" icon="fa-arrow-right" type="object" string="Buy credits" class="btn-link"/>
```

Comment:
Why not put the button only when you get the error message in the EDI?  Do not we provide for a number of free credits anyways already?  (l10n_pe_edi might have something like that already)

Reply:
this is a requirement of the onboarding team.
generally, the admin needs to buy credit before starting EDI in odoo.
also, it's pain full for the admin to find specific invoices where the error is related to low credit.

so if we only show the button if there is low credit then it's not useful for onboarding team

Reply:
I see that SMS also has buy credits.  It might just be my reaction, but it seems to create a doubt around "do I need to do it immediately or not".  Maybe some help with 200 for free, 1 cost per transaction or something...

Reply:
okay, good Idia.